src/models/student-groups/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './actions';
```

Comment:
`export { } from './actions.js';`
use * in common index.js in folder "models"

Reply:
Ok, but what is the difference between

```
export * from './actions';
```

and
```
export {
  globalLoadStudentGroups,
  loadStudentGroupsById,
  editStudentGroup,
  addStudentGroup,
  studentGroupsWatcher,
} from './actions';
``` 
?

Reply:
No difference)
According to [Sergei](https://github.com/sarhan-azizov) comments. If we want to know, which action we would like to use, we go to the folder of specific model, to index.js and check it.